apps/nextra/pages/en/build/sdks/go-sdk/building-transactions.mdx

```diff
@@ -0,0 +1,328 @@
+          ArgTypes: []aptos.TypeTag{},
+          Args: [][]byte{
+              accountBytes,
+              amountBytes,
+          },
+      }},
+  )
+  ```
+
+  <Callout type="info">
+    All arguments `Args` must be serialized to bytes before being passed in. They
+    must be serialized with [Binary Canonical Serialization (BCS)](building-transactions/bcs-format.mdx)
+  </Callout>
+
+  #### Building Options
+
+  You can customize the way your transaction executes by passing in
+  `options` when building. Some of the most commonly used options are:
+
+  1. `MaxGasAmount` - This caps the amount of gas you are willing to pay for to
+  execute this transaction.
+  2. `GasUnitPrice` - You can specify a higher than minimum price per gas to be
+  executed with higher priority by the Aptos network.
+  3. `ExpirationSeconds` - This gives a concrete time the transaction must execute
+  by or it will be canceled.
+
+  The SDK provides sensible defaults for these values if they are not specified
+  explicitly.
+
+  ### Simulate (Optional)
+
+  Every transaction on the Aptos chain has a gas fee associated with how much
+  work the network machines have to do when executing the transaction. In order
+  to estimate the cost associated with that, you can simulate transactions
+  before committing them.
+
+  <Callout type="info">
+    This simulation only requires the `PublicKey` of an account since it will
+    not impact the actual state of the ledger.
+  </Callout>
+
+  You can execute the simulation by using
+  `aptos.SimulateTransaction` like so:
+
+  ```go filename="build_a_transaction.go"
+  // 2. Simulate transaction (optional)
+  // This is useful for understanding how much the transaction will cost
+  // and to ensure that the transaction is valid before sending it to the network
+  // This is optional, but recommended
+  simulationResult, err := client.SimulateTransaction(rawTxn, alice)
+
+  // If the fee looks ok, continue to signing!
+  ```
+
+  ### Sign
+
+  Once the transaction is built and the fees seem reasonable, you can sign the
+  transaction with `rawTransaction.SignedTransaction()`. The signature must come
+  from the `sender` account.
+
+  ```go filename="build_a_transaction.go"
+  // 3. Sign transaction
+  signedTxn, err := rawTxn.SignedTransaction(alice)
+  ```
+
+  ### Submit
+
+  Now that the transaction is signed, you can submit it to the network using
+  `client.SubmitTransaction()` like so:
+
+  ```go filename="build_a_transaction.go"
+  // 4. Submit transaction
+  submitResult, err := client.SubmitTransaction(signedTxn)
+  ```
+
+  ### Wait
+
+  Finally, you can wait for the result of the transaction by using
+  `client.WaitForTransaction()` and specifying the hash of the transaction you
+  just submitted like so:
+
+  ```go filename="build_a_transaction.go"
+  // 5. Wait for the transaction to complete
+  txnHash := submitResult.Hash
+  _, err = client.WaitForTransaction(txnHash)
+  ```
+  <Fragment></Fragment>
```

Comment:
Is the fragment needed?

Reply:
I don't know... this was randomly in the TS SDK one I copied